docs/UserGuide.md

```diff
@@ -2,55 +2,62 @@
 Well, hello there! Welcome to your personal budgeting assistant, PennyWise.
 After this guide, you will understand how to use PennyWise, and spend every penny wisely. 🤑
 
+## Introduction
+
+PennyWise is a desktop application that **empowers students with the ability to make informed financial decisions**,
+by providing a **graphical analysis of their financial activities**.
+It provides a clean Graphical User Interface (GUI) for easy comprehension of expenditure and savings.
+_Interested?_ Head over to [Quick Start](#quick-start) to get started. Enjoy!
+
+<div markdown="span" class="alert alert-info">:information_source: **Info:**
+Already an expert? Jump right straight to the [features](#features) section to see how you can get the most out of PennyWise!
+</div>
+
 # Table of Contents
 <div id="top">
 </div>
 
 <!-- TOC -->
-* [Introduction](#introduction)
-* [Quick start](#quick-start)
-* [Navigating the User Guide](#navigating-the-user-guide)
+* [Purpose of Guide](#purpose-of-guide)
+* [How to use this User Guide](#how-to-use-this-user-guide)
   * [Information Box](#information-box)
   * [Tip Box](#tip-box)
   * [Danger Box](#danger-box)
   * [Formatting](#formatting)
-* [Before Using](#before-using)
+* [Quick start](#quick-start)
+* [Before Using PennyWise](#before-using-pennywise)
   * [Familiarising yourself with the interface](#familiarising-yourself-with-the-interface)
+* [Understanding the command formats](#understanding-the-command-formats)
   * [Categorising your expenses and income](#categorising-your-expenses-and-income)
   * [Guidelines on organising your expenses and income](#guidelines-on-organising-your-expenses-and-income)
 * [Features](#features)
-  * [Adding entries: `add t/ENTRY_TYPE d/DESCRIPTION a/AMOUNT da/DATE c/CATEGORY`](#adding-entries--add-tentry_type-ddescription-aamount-dadate-ccategory)
+  * [Adding entries:](#adding-entries)
```

Comment:
Given the ambiguity of `entry` for first time users, do you think we should put an explanation of "What is an entry?" like https://www.notion.so/help/category/new-to-notion as a section or FAQ 🤔 I think a section is better under "Before using" since it is quite a core part of understanding the commands. Will it be necessary? What do u think?

Reply:
yea i think this should be included under "Before Using", will push it up in another PR